protos/sources/blockexplorer/api/v1/blockexplorer.proto

```diff
@@ -65,8 +65,18 @@ message ListTransactionsRequest {
   optional string before = 2;
   // Optional cursor to paginate the request
   optional string after = 3;
+  // Deprecated: This field will be removed in favour of the fields below
+  // More filters may be added to support the legacy ones, if needed
   // Filters to apply to the request
-  map<string, string> filters = 4;
+  map<string, string> filters = 4  [deprecated = true];
+  // Transaction type filter, for command or chain event
+  repeated string tx_type = 5;
+  // Party ID filter, can be sender or receiver
+  repeated string party = 6;
+  // Sender of the transaction
+  repeated string sender = 7;
+  // Receiver of the transaction
```

Comment:
Dunno if it's maybe worth being a bit more explicit about this as there isn't really a 'receiver' for a transaction; but I know what you mean.. perhaps something like 'In the case of a deposit.. etc'. Maybe that's too wordy

Reply:
Ah I see. We could say something like "where it applies" ?